src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1677,7 +1679,7 @@ Unable\ to\ generate\ new\ library=Unable to generate new library
 Open\ console=Open console
 Use\ default\ terminal\ emulator=Use default terminal emulator
 Execute\ command=Execute command
-Note\:\ Use\ the\ placeholder\ %0\ for\ the\ location\ of\ the\ opened\ library\ file.=Note: Use the placeholder %0 for the location of the opened library file.
+Note\:\ Use\ the\ placeholder\ %DIR%\ for\ the\ location\ of\ the\ opened\ library\ file.=Note: Use the placeholder %DIR% for the location of the opened library file.
```

Comment:
This is a parameterized localization string, e.g. the %0 works similar as in String.format as argument and is usually used to indicate values that should not be translated

Not sure if it is possible to use them in the fxml directly

Reply:
It showed up fine, I did not see any complaints from the compiler nor from the jre. The original line was
>  new Label(Localization.lang("Note: Use the placeholder %0 for the location of the opened library file.", "%DIR") )
The other option I thought of would be to put this in the initialize-method as setValue. If you want so, I can change it, but I don't think that this would be a good solution, as everything else is completly written in fxml.